docs/manual-EN/statement-syntax/data-definition-statements/create-tag-edge-syntax.md

```diff
@@ -1,28 +1,31 @@
-### Create TAG / EDGE Syntax
+# Create TAG / EDGE Syntax
 
 ```
-CREATE {TAG | EDGE} tag_name|edge_name
-    (create_definition, ...)
+CREATE {TAG | EDGE} <tag_name> | <edge_name>
+    (<create_definition>, ...)
     [tag_edge_options]
   
-create_definition:
-    prop_name data_type
-    
-tag_edge_options:
-    option [, option ...]
+<create_definition> ::=
+    <prop_name> <data_type>
+
+<tag_edge_options> ::=
+    <option> [, <option> ...]
 
-option:
-    TTL_DURATION [=] ttl_duration
-    | TTL_COL [=] prop_name
+<option> ::=
+    TTL_DURATION [=] <ttl_duration>
+    | TTL_COL [=] <prop_name>
+    | DEFAULT <default_value>
 ```
 
 Nebula's graph schema is composed of tags and edges, either of which may have properties. `CREATE TAG` statement defines a tag with the given name. `CREATE EDGE` statement is to define an edge type.
```

Comment:
`CREATE TAG` statement defines a tag with the given name while `CREATE EDGE` defines an edge type.